python/app/dialog.py

```diff
@@ -1430,11 +1435,11 @@ def load_sort_data(self, field, sort_action, actions_list, **sort_order):
 
         # Set checked the current sort order in the Menu
         if sort_order == "asc":
-            actions_list[3].setChecked(True)
-            actions_list[4].setChecked(False)
+            actions_list[0].setChecked(True)
+            actions_list[1].setChecked(False)
         elif sort_order == "desc":
-            actions_list[4].setChecked(True)
-            actions_list[3].setChecked(False)
+            actions_list[0].setChecked(False)
+            actions_list[1].setChecked(True)
```

Comment:
I moved the sort order to the top, it was shown at the bottom of the list. This brings it inline with the web UI.